src/main/java/org/broadinstitute/hellbender/tools/spark/sv/SVVariantCallerInternal.java

```diff
@@ -0,0 +1,338 @@
+    }
+
+    // TODO: this is looking at one chimeric alignment, i.e. a pair of alignment records at a time, potentially making calling complex variant difficult
+    /**
+     * Second step in calling variants: produce a single {@link BreakpointAllele} from a single {@link ChimericAlignment} of a contig.
+     *
+     * @param breakpointIdAndAssembledBreakpoint    the inner Tuple2 is a pair of (asmID, contigID) for the contig that generated this chimeric alignment
+     */
+    static Tuple4<BreakpointAllele, String, String, ChimericAlignment> keyByBreakpointAllele(final Tuple2<Tuple2<String, String>, ChimericAlignment> breakpointIdAndAssembledBreakpoint) {
+        final ChimericAlignment chimericAlignment = breakpointIdAndAssembledBreakpoint._2();
+        // TODO: 11/4/16 INSDEL calling should add another kind of BreakpointAllele here
+        final BreakpointAllele breakpointAllele = chimericAlignment.involvesStrandSwitch() ? new BreakpointAlleleInversion(chimericAlignment) : new BreakpointAllele(chimericAlignment);
+        return new Tuple4<>(breakpointAllele, breakpointIdAndAssembledBreakpoint._1._1, breakpointIdAndAssembledBreakpoint._1._2, chimericAlignment);
+    }
+
+    /**
+     * Third step in calling variants: produce a VC from a {@link BreakpointAllele} (consensus among different assemblies if they all point to the same breakpoint).
+     *
+     * @param assembledBreakpointsPerAllele     consensus among different assemblies if they all point to the same breakpoint
+     * @param broadcastReference                broadcasted reference
+     * @throws IOException                      due to read operations on the reference
+     */
+    static VariantContext getVariantFromBreakpointAlleleAlignments(final Tuple2<BreakpointAllele, Iterable<Tuple3<String, String, ChimericAlignment>>> assembledBreakpointsPerAllele,
+                                                                   final Broadcast<ReferenceMultiSource> broadcastReference) throws IOException {
+
+        final BreakpointAllele breakpointAllele = assembledBreakpointsPerAllele._1;
+        final String contig = breakpointAllele.leftAlignedLeftBreakpoint.getContig();
+        final int start = breakpointAllele.leftAlignedLeftBreakpoint.getStart();
+        final int end = breakpointAllele.leftAlignedRightBreakpoint.getStart();
+        VariantContextBuilder vcBuilder = new VariantContextBuilder().chr(contig).start(start).stop(end);
+
+        vcBuilder = vcBuilder.alleles(produceAlleles(broadcastReference.getValue(), contig, start, end))
+                             .id(produceVariantId(breakpointAllele));
+        vcBuilder = updateAttributes(vcBuilder, breakpointAllele, assembledBreakpointsPerAllele._2, start, end);
+
+        return vcBuilder.make();
+    }
+
+    // -----------------------------------------------------------------------------------------------
+    // Group 1: from contig alignments to breakpoints
```

Comment:
I prefer (habits from using text editors not IDEs) blocking codes into logic units for my own reading.
Removed.